go/vt/vtgate/executor.go

```diff
@@ -259,6 +264,7 @@ func (e *Executor) handleExec(ctx context.Context, safeSession *SafeSession, sql
 		logStats.BindVariables = bindVars
 		result, err := e.destinationExec(ctx, safeSession, sql, bindVars, dest, destKeyspace, destTabletType, logStats)
 		logStats.ExecuteTime = time.Now().Sub(execStart)
+		queriesRouted.Add("ShardDirect", int64(logStats.ShardQueries))
```

Comment:
Ah now I see what you mean by `queriesProcessed` vs `queriesRouted`. `queriesProcessed` is queries going into vtgate and `queriesRouted` is queries going out of vtgate to vttablet?

Reply:
Yep! I'm not in love with the names so if we can come up with more intuitive options, I'd be happy to change accordingly.

Reply:
The other similar metric to `queriesRouted` is `vtgate_vttablet_call` (connections from vtgate to vttablets, weirdly), and `vtgate_api`, neither of which are very consistent with each other. QueriesProcessed and QueriesRouted seems fine to me.

Reply:
Yep. The difference is that the `vtgate_vttablet_call` includes all API calls and not just execute queries (though in practice with v3 I think that's basically all the vtgate would do), and of course these aren't tagged by plan type.